licenses/bin/hamcrest-date-2.0.4.txt

```diff
@@ -0,0 +1 @@
+According to https://github.com/eXparity/hamcrest-date Hamcrest-date is under BSD License
```

Comment:
>According to https://github.com/eXparity/hamcrest-date

I'm not sure https://github.com/eXparity/hamcrest-date would always represent licensing state for hamcrest-date-2.0.4

Should some more specific link be used here? For instance, https://search.maven.org/#artifactdetails%7Corg.exparity%7Chamcrest-date%7C2.0.4%7Cjar or https://github.com/eXparity/hamcrest-date/blob/hamcrest-date-2.0.4/pom.xml#L18-L24